product_input_interface/allowed_products.py

```diff
@@ -0,0 +1,28 @@
+class WarehouseProducts:
+    __ALLOWED_PRODUCTS_TYPE = ["helmet", "rifle", "medkit"]
+    __MAX_PRODUCT_NAME_LENGTH = 60
+    __MIN_PRODUCT_NAME_LENGTH = 1
+
+    def print_allowed_product_types(self):
+        print("Allowed product types:")
+        for item in self.__ALLOWED_PRODUCTS_TYPE:
+            print(f"- {item}")
+
+    def check_product_type(self, product_type: str):
+        if product_type in self.__ALLOWED_PRODUCTS_TYPE:
+            return True
+        else:
+            print("Product type was not valid.")
```

Comment:
that's "pure logic` function - please avoid side-effects in such places by using `print` - higher code should decide what to do when `False` is returned